cg/meta/report/balsamic.py

```diff
@@ -177,3 +184,10 @@ def get_template_name(self) -> str:
         """Retrieves the template name to render the delivery report"""
 
         return Pipeline.BALSAMIC + "_report.html"
+
+    def get_scout_file_tags(self, scout_tag: str) -> Optional[list]:
+        """Retrieves BALSAMIC uploaded to scout file tags"""
+
+        tags = BALSAMIC_CASE_TAGS.get(scout_tag)
+
+        return list(tags) if tags else None
```

Comment:
this sub is repeated in the balsamic, balsamic_umi, and MIP met/report, could it be moved to a base function to be reused by sending in the "X_CASE_TAGS" to be returned?